test/regress/cli/regress1/fmf/issue10943.smt2

```diff
@@ -0,0 +1,6 @@
+(set-logic ALL)
```

Comment:
Shouldn't there be a status for this benchmark?

Reply:
Forgot to add it, will fix.